--- mame-mame0249/src/lib/netlist/plib/ptypes.h.orig	2022-10-27 00:12:58.000000000 +0200
+++ mame-mame0249/src/lib/netlist/plib/ptypes.h	2024-09-24 22:34:28.500092609 +0200
@@ -10,6 +10,7 @@
 
 #include "pconfig.h"
 
+#include <cstdint>
 #include <limits>
 #include <string>
 #include <type_traits>
--- mame-mame0249/src/osd/modules/monitor/monitor_module.h.orig	2022-10-27 00:12:58.000000000 +0200
+++ mame-mame0249/src/osd/modules/monitor/monitor_module.h	2024-09-25 06:22:41.836749799 +0200
@@ -8,6 +8,7 @@
 #ifndef MAME_OSD_MODULES_MONITOR_MONITORMODULE_H
 #define MAME_OSD_MODULES_MONITOR_MONITORMODULE_H
 
+#include <cstdint>
 #include <vector>
 
 #include "modules/osdmodule.h"
--- mame-mame0249/src/osd/modules/monitor/monitor_common.h.orig	2022-10-27 00:12:58.000000000 +0200
+++ mame-mame0249/src/osd/modules/monitor/monitor_common.h	2024-09-25 06:23:06.383562606 +0200
@@ -8,6 +8,7 @@
 #define __MONITOR_COMMON_H__
 
 #include "monitor_module.h"
+#include <cstdint>
 #include <map>
 
 //============================================================
--- mame-mame0249/src/frontend/mame/ui/slider.h.orig	2022-10-27 00:12:58.000000000 +0200
+++ mame-mame0249/src/frontend/mame/ui/slider.h	2024-09-25 06:24:44.315236300 +0200
@@ -14,6 +14,7 @@
 
 #pragma once
 
+#include <cstdint>
 #include <functional>
 #include <string>
 
