--- mame-0.167s/scripts/genie.lua.orig	2015-10-28 09:47:14.000000000 +0100
+++ mame-0.167s/scripts/genie.lua	2015-11-25 21:54:31.968230955 +0100
@@ -1041,7 +1041,6 @@
 				buildoptions {
 --					"-D__USE_MINGW_ANSI_STDIO=1", -- required or lua won't compile linux ignores this but Windows needs it
 					"-freport-bug",
-					"-D_GLIBCXX_USE_CXX11_ABI=0", -- does not seem to matter in linux, mingw needs to link printf,etc
 --					"-DNO_MEM_TRACKING",          -- must comment out for mingw GCC 5.2 pedantic or get new/delete redef error
 -- next two should work, but compiler complains about end conditions that are int when loop variable is unsigned. maybe these can be fixed
 --					"-funsafe-loop-optimizations",
